deployment/helm/values.yaml

```diff
@@ -45,15 +45,13 @@ exposedServiceType: NodePort # ClusterIP | NodePort | LoadBalancer
 #   - HUB_WEBAPP_HOST
 #   - HUB_CFSSL_HOST
 #   - ALERT_SERVER_PORT
-# Format: array of strings delimited by commas; each string has the format "KEY:VALUE" e.g. ["HUB_WEBAPP_HOST:localhost", "ALERT_SERVER_PORT:443"]
-environs: []
+environs: {}
```

Comment:
can we just add examples here in the comments similar to blackduck: 
# if you are setting the value using set flag in helm command, do --set environs.* = ""; i.e.: --set environs.HUB_POSTGRES_CONNECTION_ADMIN="blackduck@pg-server-name"

Reply:
done!